ocaml/xapi/create_misc.mli

```diff
@@ -41,3 +41,4 @@ val make_software_version : __context:Context.t -> (string * string) list
 val create_host_cpu : __context:Context.t -> unit
 val create_pool_cpuinfo : __context:Context.t -> unit
 val create_chipset_info : __context:Context.t -> unit
+val create_patches_requiring_reboot_info : __context:Context.t -> host:[`host] Ref.t -> [ `pool_patch ] Ref.t list
```

Comment:
I think it would be good to move the DB set call into this new function, and have the return type be `unit`, because I don't think that the callers need the value (they just need it to go into the DB). You can then call it `set_patches_requiring_reboot`.